compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -1427,6 +1427,13 @@ fn collect_miri<'tcx>(tcx: TyCtxt<'tcx>, alloc_id: AllocId, output: &mut MonoIte
                 output.push(create_fn_mono_item(tcx, fn_instance, DUMMY_SP));
             }
         }
+        GlobalAlloc::Vtable(ty, trait_ref) => {
+            // FIXME(RJ) no ideas if this is correct. There is this nice
+            // `create_mono_items_for_vtable_methods` method but I wouldn't know how to call it from
+            // here. So instead we just generate the actual vtable and recurse.
+            let alloc_id = tcx.vtable_allocation((ty, trait_ref));
+            collect_miri(tcx, alloc_id, output)
```

Comment:
I don't have any clue what I should do here, whether I should call `create_mono_items_for_vtable_methods`, and if yes how to do that.

Reply:
I don't think you need to do so here as I think `collect_miri` would already create mono items for all vtable methods referenced by the vtable.

Reply:
I think it makes sense, but I am not an expert on the monomorphization collector.

Reply:
Okay I will remove the FIXME comment then.